bite-size-notes/week-0-b-test.md

```diff
@@ -0,0 +1,9 @@
+## new notes
+
+Cats are cute.
+
+Boba is yummy.
+
+Another photo of a cat:
```

Comment:
But this is the same picture as before. Can you find a different photo to make the argument more convincing.

Reply:
I agree. I found another cute photo.